.config/i3/config

```diff
@@ -0,0 +1,183 @@
+# i3 config file (v4)
```

Comment:
These are needed for the linux install but not so much for mac - they'll not be symlinked by the setup playbook